nilearn/plotting/_utils.py

```diff
@@ -6,14 +6,17 @@
 import numpy as np
 from matplotlib.colors import LinearSegmentedColormap
 
-from nilearn.surface import (
-    PolyMesh,
-    SurfaceImage,
-)
-from nilearn.surface.surface import combine_hemispheres_meshes, get_data
+
+def engine_warning(engine):
+    warning = (
+        f"'{engine}' is not installed. To be able to use '{engine}' as "
+        "plotting engine for 'nilearn.plotting' package:\n"
+        " pip install 'nilearn[plotting]'"
+    )
+    warn(warning)
 
 
-def save_figure_if_needed(fig, output_file):
+def save_figure_if_needed(fig, output_file, bbox_inches=None):
```

Comment:
It would be good to update this later to use SurfaceFigure interface. And implementing an equivalent of PlotlySurfaceFigure for matplotlib might be an option.